packages/wekan-oidc/oidc_server.js

```diff
@@ -180,16 +181,16 @@ var getToken = function (query) {
 
 // Function to fetch user information from the OIDC service
 var getUserInfo = function (accessToken) {
-    var debug = process.env.DEBUG === 'true';
+    var debug = process.env.DEBUG === 'true';  // Check if debug mode is enabled
     var config = getConfiguration();
-    var serverUserinfoEndpoint = config.userinfoEndpoint.includes("https://") ?
+    var serverUserinfoEndpoint = config.userinfoEndpoint.includes("https://") ? 
         config.userinfoEndpoint : config.serverUrl + config.userinfoEndpoint;
 
     var response;
     try {
         var getOptions = {
             headers: {
-                "User-Agent": "Meteor",
+                "User-Agent": userAgent,
```

Comment:
@walster001 

What is happening here?

Reply:
AI Modified the useragent to be "Meteor" instead of it's original variable "userAgent" according to the last submitted version of oidc_server.js
However, is userAgent even set as a variable or would it benefit from being set to meteor permanently?